.github/workflows/gaia.yml

```diff
@@ -64,31 +60,31 @@ jobs:
     - name: Build Ir on Experimental Store
       run: |
         source ${HOME}/.bashrc
-        cd ${GITHUB_WORKSPACE}/research/query_service/ir/compiler
+        cd ${GITHUB_WORKSPACE}/interactive_engine/executor/ir/compiler
         make build
 
     - name: Build Ir on Vineyard Store
       run: |
         source ${HOME}/.bashrc
         cd ${GITHUB_WORKSPACE} && make install
-        cd ${GITHUB_WORKSPACE}/interactive_engine/executor
-        cargo fmt -- --check
+      # cd ${GITHUB_WORKSPACE}/interactive_engine/executor
+      # cargo fmt -- --check
 
     - name: Build Ir on Groot Store
       run: |
         source ${HOME}/.bashrc
         cd ${GITHUB_WORKSPACE}/interactive_engine
-        mvn clean install -DskipTests -Pv2 -Drust.compile.mode=debug --quiet
+        mvn clean install -DskipTests -Pv2 -Drust.compile.mode=release --quiet
```

Comment:
Why change the mode to `release`.
I think it's ok to use debug mode in CI environment and it can speed up the compiling.

Reply:
I change this to `release` as all build else (e.g., vineyard, exp_store) is build as `release` for now. If so, we can change all as `debug`.

Reply:
I revert this and also change building on exp_store as `debug`.

Reply:
`debug` mode can speed up compiling, but result in slow testing.

Reply:
So which one is faster? debug compile + run or release compile + run?

Reply:
seems like release compile + run is faster